### ISSUE TITLE
Maui RC1 - "System.NotSupportedException: Must call AddTerm AddSeparator in that order"

### ISSUE THREAD

**Issue by Mafyou** [dotnet/maui#6489]
### Description

I installed the last update for Maui (https://github.com/dotnet/maui). When I add an image, it makes an error: "error MAUI0000: System.NotSupportedException: Must call AddTerm AddSeparator in that order"
https://stackoverflow.com/questions/71868449/maui-rc1-system-notsupportedexception-must-call-addterm-addseparator-in-that

### Steps to Reproduce

Update your VS2022 to the last update.
Create a Maui Mobile
Add Image to the "Resources/Images"
Put it the kind to "MauiImage"
Try to compile.

### Version with bug

Release Candidate 1 (current)

### Last version that worked well

Preview 14

### Affected platforms

I was *not* able test on other platforms

### Affected platform versions

All

### Did you find any workaround?

Go Release instead of Debug and it works.
Nah, it took cache and if the solution is cleaned, it break again with the same error.

### Relevant log output

```shell
error MAUI0000: System.NotSupportedException: Must call AddTerm AddSeparator in that order
```

**Comment by XamlTest** [comment]
Verified not repro on Visual Studio Enterprise 17.3.0 Preview 1.0 [32414.199.main]. Sample Project: [6489.zip](https://github.com/dotnet/maui/files/8558909/6489.zip)

**Comment by Mafyou** [comment]
In human language?

**Comment by Mafyou** [comment]
@XamlTest, try in SVG then.

**Comment by Redth** [comment]
This should be fixed in the upcoming VS 17.3 Preview 1 release.
